test/enigma_test.rb

```diff
@@ -93,4 +93,26 @@ def test_it_can_generate_keys
     @enigma.stubs(:generate_number).returns(6)
     assert_equal "00006", @enigma.generate_key
   end
+
+  def test_it_can_read_from_txt_file
+    filepaths = ["./text_files/test_message.txt", "./text_files/test_message_encrypted.txt"]
+    @enigma.stubs(:user_input).returns(filepaths)
+    expected = "Hello World!"
+    assert_equal expected, @enigma.read_from_txt
+    assert_equal expected, @enigma.read_from_txt("./text_files/test_message.txt")
+  end
+
+  def test_it_can_write_to_txt
+    filepaths = ["./text_files/test_message.txt", "./text_files/test_message_write.txt"]
+    @enigma.stubs(:user_input).returns(filepaths)
+    assert_equal false, File.exists?("./text_files/test_message_write.txt")
+    @enigma.write_to_txt("hello world")
+    assert_equal true, File.exists?("./text_files/test_message_write.txt")
+    expected = "hello world"
+    assert_equal expected, @enigma.read_from_txt("./text_files/test_message_write.txt")
```

Comment:
This is one such situation where the default value makes it easier to test as I can have it read a file that is not the same as the one from user_input.first.